contracts/CryptoChampions.sol

```diff
@@ -812,4 +836,35 @@ contract CryptoChampions is ICryptoChampions, AccessControl, ERC1155, VRFConsume
         grantRole(ROLE_GAME_ADMIN, address(priceWar));
         priceWar.startGame();
     }
+
+    /// @notice Transfers in game currency tokens from one address to another
+    /// @param to The receiving address
+    /// @param amount The amount to transfer
+    function transferInGameTokens(address to, uint256 amount) external override {
```

Comment:
Is this function necessary? safeTransferFrom is already public we could just access it directly

Reply:
So the reason for this is that the weather wars factory contract is talking to the `ICryptoChampions` interface instead of the `CryptoChampions` contract so it can't access the `safeTransferFrom` function.